settings.gradle.kts

```diff
@@ -6,22 +12,23 @@ pluginManagement {
     }
 
     plugins {
-        val kotlinVersion = extra["kotlin.version"] as String
-        val agpVersion = extra["agp.version"] as String
-        val composeVersion = extra["compose.version"] as String
```

Comment:
Why do you remove extra["kotlin.version"] ?
is it because of the newer Gradle 8.0.2 ?

Reply:
Putting versions into a `properties` file was a compromise between convinience for users and convinience to test templates on CI.

It was refactored multiple times, and probably will be refactored in the future, as the recommended by Gradle way is to store version in the Gradle Catalog's `toml` file.

But it should be done at once for all our templates/examples.

Or after a decision to treat templates as your own thing, and make copies for CI testing. This decision looks right, but decisions are hard :).